### PR TITLE
group 7 changes

### PR DESCRIPTION
**Pull request by N-Devin**

### CODE REVIEW

**Comment by abarcomb-work** [comment]
Pull requests should contain useful information about the reasoning behind the changes for the repository maintainer to enable them to prioritize PRs. This is part of the collaborative workflow.

**Comment by abarcomb-work** [comment]
The PR should have been made to the ensf400-ws24 branch, not to main.

File: src/.DS_Store
Comment (growlf) [inline src/.DS_Store:1]: This file should not be added - use the `.ignore*` files to preclude accidental additions when on a Mac system.

File: .DS_Store
Comment (growlf) [inline .DS_Store:1]: This file should not be added - use the .ignore* files to preclude accidental additions when on a Mac system.

**Comment by growlf** [comment]
This PR seems to be comprised of code and changes that already exist in other branches or files, and so does not contribute anything new or valuable.

**Pull request closed without merging**